packages/hint/docs/user-guide/concepts/parsers.md

```diff
@@ -16,40 +16,35 @@ configuration file:
 
 ## List of official `parser`s
 
+* [`css`][@hint/parser-css]: A `CSS` parser built on top of
```

Comment:
FYI, I also updated this list in my typed events PR:
https://github.com/webhintio/hint/pull/1397/files#diff-ac4792357afd3a5de0c4ab23c27aee72